Common/GPU/Vulkan/thin3d_vulkan.cpp

```diff
@@ -1298,15 +1307,27 @@ void VKContext::UpdateBuffer(Buffer *buffer, const uint8_t *data, size_t offset,
 	memcpy(buf->data_ + offset, data, size);
 }
 
-void VKContext::BindTextures(int start, int count, Texture **textures) {
+void VKContext::BindTextures(int start, int count, Texture **textures, TextureBindFlags flags) {
 	_assert_(start + count <= MAX_BOUND_TEXTURES);
 	for (int i = start; i < start + count; i++) {
+		_dbg_assert_(i >= 0 && i < MAX_BOUND_TEXTURES);
 		boundTextures_[i] = static_cast<VKTexture *>(textures[i - start]);
-		boundImageView_[i] = boundTextures_[i] ? boundTextures_[i]->GetImageView() : GetNullTexture()->GetImageView();
+		boundTextureFlags_[i] = flags;
+		if (boundTextures_[i]) {
+			// NOTE: These image views are actually not used, it seems - they get overridden in GetOrCreateDescriptorSet
```

Comment:
Well, I think (?) it's used for `BindFramebufferAsTexture()`?  Maybe it's best to set it to nullptr here.  It feels like it'd be better to either have `boundTextures_[binding] != nullptr`, `boundImageView_[binding] != nullptr` or both nullptr, but not both set.

I guess BOUND_TEXTURE0_IMAGEVIEW/BOUND_TEXTURE1_IMAGEVIEW (BoundFramebufferTexture) potentially uses these, though?

-[Unknown]

Reply:
Yeah I'm wrong here, we need it for BOUND*. So I'll change the comment.